transport/src/iosMain/kotlin/com/genesys/cloud/messenger/transport/util/TokenStoreImpl.kt

```diff
@@ -1,11 +1,10 @@
 package com.genesys.cloud.messenger.transport.util
 
-import com.liftric.kvault.KVault
 import platform.Foundation.NSUUID
 
 internal class TokenStoreImpl(storeKey: String) : TokenStore {
 
-    private val store = KVault(storeKey)
+    private val store = Vault(storeKey)
     override val token: String
         get() = store.string(TOKEN_KEY) ?: NSUUID().UUIDString().also {
             store.set(TOKEN_KEY, it)
```

Comment:
It looks like this `set()` returns a bool for success/failure. If the set fails would it be worthwhile to log the failure?

Reply:
PR updated.